controller/stats.controller.js

```diff
@@ -3,37 +3,45 @@ const NotFoundError = require("../errors/NotFound.errors");
 const ServerError = require("../errors/internal.errors");
 const ExperimentRepository = require("../repositories/experiment.repository");
 const UserRepository = require("../repositories/user.repository");
+const GoalRepository = require("../repositories/goal.repository")
 const {mongoose} = require("mongoose");
 
 const userPercentageVariantByExperiment = async (experimentID, variant) => {
     return (await UserRepository.numUsersByExperimentIdAndVariant(experimentID, variant) / await UserRepository.numUsersByExperimentId(experimentID)) * 100;
 }
 
-const getStatistics = async (req, res) => {
+const calculateSuccessPercentage = (successCount, callCount) => {
```

Comment:
where did you get success count and callCount from?

Reply:
this is Ilays code im not sure
but callCount is in the experiment schema